fashion-example-query/fashion/app.py

```diff
@@ -149,13 +150,15 @@ def query(num_doc, target: dict):
 
 def config(task):
     parallel = 2 if task == 'index' else 1
+    shards = 1
     os.environ['RESOURCE_DIR'] = resource_filename('jina', 'resources')
-    os.environ['SHARDS'] = str(1)
+    os.environ['SHARDS'] = str(shards)
     os.environ['PARALLEL'] = str(parallel)
-    os.environ['WORKDIR'] = './workspace'
-    os.makedirs(os.environ['WORKDIR'], exist_ok=True)
+    os.environ['HW_WORKDIR'] = './workspace'
+    os.makedirs(os.environ['HW_WORKDIR'], exist_ok=True)
```

Comment:
Since we anyhow touch the environment variables, can we please prefix them with `JINA_`? Thanks a lot.

Reply:
Sure, makes sense

Reply:
Done